build/params_testground.go

```diff
@@ -0,0 +1,6 @@
+// +build testground
+
+package build
+
+const BlockDelay = 1
+const PropagationDelay = 6
```

Comment:
What we probably want is to make these vars so we can adjust them at will from testground tests. And we probably need to do something similar in specs-actors will all their default settings.

Reply:
It is not trivial to make them `var`s (see updated PR description). Also it is not clear we HAVE to do that right now, so for now I opted to experiment with a build flag.

Reply:
Yeah, I mean making them vars through a build tag.